webapp/src/components/ParcelPreview/ParcelCanvas/ParcelCanvas.js

```diff
@@ -417,52 +402,29 @@ export default class ParcelPreview extends React.PureComponent {
   getParcelAttributes = (x, y) => {
     const parcelId = buildCoordinate(x, y)
 
-    if (!this.cache[parcelId]) {
-      const { wallet, parcels, districts, publications } = this.props
-      const parcel = parcels[parcelId]
-      const publication = getOpenPublication(parcel, publications)
-
-      const color = getTextColor(parcelId, x, y, parcels, publications, wallet)
-      const backgroundColor = getColor(
-        parcelId,
-        x,
-        y,
-        parcels,
-        publications,
-        wallet
-      )
-      const label = getLabel(
-        parcelId,
-        x,
-        y,
-        parcels,
-        publications,
-        wallet,
-        districts
-      )
-      const description = getDescription(
-        parcelId,
-        x,
-        y,
-        parcels,
-        publications,
-        wallet
-      )
-
-      this.cache[parcelId] = {
-        id: parcelId,
-        publication,
-        connectedLeft: !!(parcel && parcel.connectedLeft),
-        connectedTop: !!(parcel && parcel.connectedTop),
-        connectedTopLeft: !!(parcel && parcel.connectedTopLeft),
-        color,
-        backgroundColor,
-        label,
-        description
-      }
+    const { wallet, parcels, districts, publications, estates } = this.props
```

Comment:
Why are we not caching anymore here? (the `this.cache[parcelId] = asset` is still there)